OpenEdXMobile/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -176,6 +176,14 @@ public void onBackPressed() {
         }
     }
 
+    /**
+     * It will add the slide drawer in the activity's layout.
+     * <p>
+     * For addition, {@link DrawerLayout} with id {@link R.id#drawer_layout R.id.drawer_layout} will
+     * be search out in activity's layout, if it exists {@link NavigationFragment navigation drawer}
+     * will be replaced by the container having id {@link R.id#slider_menu R.id.slider_menu}.
```

Comment:
```java
    /**
     * It will add the slide drawer in the activity's layout.
     * <p>
     * For addition, {@link DrawerLayout} with id {@link R.id#drawer_layout R.id.drawer_layout} will
     * be searched in activity's layout, if it exists, the {@link NavigationFragment Navigation Drawer}
     * will be replaced by the container having id {@link R.id#slider_menu R.id.slider_menu}.
     * </p>
     */
```